main.sh

```diff
@@ -162,22 +162,29 @@ case $choice in
         fi
 
         if [[ $choice == "load" || $1 == "load" ]]; then
-
+            
             if [[ $choice == "load" ]]; then
+            echo -e "\e[0m Available configs: \e[0m"
+            for entry in "profiles"/*.txt
+            do
+                if [ -f "$entry" ];then
+                    echo -e "\e[36m   $(basename $entry) \e[0m"
```

Comment:
This prints `.txt` with the config name as well.
Showing the filename, like `sample` should be preferred instead of showing `sample.txt`, because then the user would enter the extension too, which becomes `sample.txt.txt` in the code following this.

This might help resolve this.
```bash
echo $entry | cut -d\. -f1 | xargs basename
```